src/signals/incident/definitions/wizard-step-2-vulaan/wegen-verkeer-straatmeubilair.js

```diff
@@ -20,6 +20,24 @@ const intro = {
 };
 
 export const controls = {
+  extra_kerstverlichting: {
+    meta: {
+      label: '',
+      type: 'alert',
+      value:
+        'Doet de sierverlichting in een winkelstraat het niet? Of hebt u last van de kerstverlichting? Neem dan contact op met de winkeliersvereniging. De gemeente gaat hier helaas niet over.',
+      ifAllOf: {
+        subcategory: [
+          'lantaarnpaal-straatverlichting',
+          // This element will be enabled each year near the christmass.
+          // Comment/Uncomment next line to show/hide it.
+          // 'do-not-show-this-message',
```

Comment:
Wouldn't it be cleaner to comment out the entire block when necessary? Otherwise you have to interpret the subcategory values to determine why the alert is (not) being shown.